client/.eslintrc

```diff
@@ -15,6 +15,7 @@
   "rules": {
     // suppress errors for missing 'import React' in files
    "react/react-in-jsx-scope": "off",
-   "react/prop-types": "off"
+   "react/prop-types": "off",
+   "prettier/prettier": ["off", { "singleQuote": true }]
```

Comment:
I believe "off" actually turns off prettier, which is not what we want.

Reply:
Oh does it? Well nothing else is working for me atm.